core/deploy.go

```diff
@@ -26,7 +26,7 @@ func (c *CLab) Deploy(ctx context.Context, options *DeployOptions) ([]runtime.Ge
 
 	log.Debugf("lab Conf: %+v", c.Config)
 	if options.reconfigure {
-		_ = c.Destroy(ctx, uint(len(c.Nodes)), true)
+		_ = c.destroy(ctx, uint(len(c.Nodes)), true)
```

Comment:
the "old" destroy method is now private and called by the "new" one that does all the logic that cmd used to do.